tests/pyspark/test_pyspark_config.py

```diff
@@ -326,3 +335,25 @@ class TestSchema(DataFrameModel):
             dict(output_dataframemodel_df.pandera.errors["SCHEMA"])
             == expected_dataframemodel["SCHEMA"]
         )
+
+    @pytest.mark.parametrize("cache_enabled", [True, False])
+    @pytest.mark.parametrize("unpersist_enabled", [True, False])
+    # pylint:disable=too-many-locals
+    def test_pyspark_cache_settings(
```

Comment:
Test only if the new values for `Config` are valid, not their effects.

Reply:
Does this test actually test anything useful?

Reply:
It was added to do code coverage tests stop complaining about lack of coverage in the `Config` object. Not very useful but can avoid future changes in behavior, if someone changes it.